index.html

```diff
@@ -2,12 +2,126 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
-
     <link href="styles.css" rel="stylesheet" />
-
-    <title>Rocket Academy</title>
+    <title>Ian Lau's Portfolio</title>
   </head>
   <body>
-    <h1>Hello, world!</h1>
+    <section id="head">
+      <h1>Welcome to Ian Lau's Portfolio Page</h1>
+      <img src="img/photo.jpg" class="photo-img" />
+    </section>
+    <nav class="navi">
+      <div class="navi-link">
+        <h3><a href="#about-me">About Me</a></h3>
+      </div>
+      <div class="navi-link">
+        <h3><a href="#project">Project</a></h3>
+      </div>
+      <div class="navi-link">
+        <h3><a href="#contact">Contact</a></h3>
+      </div>
+    </nav>
+    <section class="content">
+      <div id="about-me">
```

Comment:
can we give the parent section/main the id for styling and potentially remove the extra div?